rulesengine_client/tests/test_models.py

```diff
@@ -195,6 +202,18 @@ def test_protocol_applies(self):
             'block')
         self.assertEqual(rule.protocol_applies('https'), True)
 
+    def test_subdomain_applies(self):
+        rule = Rule(
+            'http://(com,example,)',
+            'block',
+            subdomain='www')
+        self.assertEqual(rule.subdomain_applies('www'), True)
```

Comment:
nit: these could be `assert(True|False)`

Reply:
agreed, and I've updated.